internal/controller/pluggableprovider_controller.go

```diff
@@ -72,44 +81,141 @@ func (r *PluggableProviderReconciler) updateStatus(ctx context.Context, pprov *k
 	if err != nil {
 		return err
 	}
+
 	if exposedProviders == "" {
 		return nil
 	}
 
 	pprov.Status.ExposedProviders = exposedProviders
 	if err := r.Client.Status().Update(ctx, pprov); err != nil {
-		return fmt.Errorf("failed to update PluggableProvider %s/%s status: %w", pprov.Namespace, pprov.Name, err)
+		return fmt.Errorf("failed to update PluggableProvider %q status: %w", pprov.Name, err)
 	}
 
 	return nil
 }
 
-func (r *PluggableProviderReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Result, err error) {
+func (r *PluggableProviderReconciler) update(ctx context.Context, pprov *kcm.PluggableProvider) (_ ctrl.Result, err error) {
 	l := ctrl.LoggerFrom(ctx)
-	l.Info("PluggableProvider reconcile start")
+	l.Info("PluggableProvider object event")
 
-	pprov := &kcm.PluggableProvider{}
-	if err := r.Get(ctx, req.NamespacedName, pprov); err != nil {
-		return ctrl.Result{}, client.IgnoreNotFound(err)
+	if err := r.addLabels(ctx, pprov); err != nil {
+		l.Error(err, "PluggableProvider adding labels error")
+		return ctrl.Result{RequeueAfter: r.syncPeriod}, err
 	}
 
-	if err := r.addLabels(ctx, pprov); err != nil {
-		return ctrl.Result{}, err
+	if err := r.updateStatus(ctx, pprov); err != nil {
+		l.Error(err, "PluggableProvider update status error")
+		return ctrl.Result{RequeueAfter: r.syncPeriod}, err
+	}
+
+	return ctrl.Result{}, nil
+}
+
+func (r *PluggableProviderReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	l := ctrl.LoggerFrom(ctx)
+	l.V(6).Info("PluggableProvider reconcile start")
```

Comment:
```suggestion
	l.V(1).Info("PluggableProvider reconcile start")
```

We do not use `6` level, either `0` or `1`.